lib/chat_api/conversations.ex

```diff
@@ -223,6 +223,14 @@ defmodule ChatApi.Conversations do
     archive_conversation(conversation)
   end
 
+  def list_conversations_to_archive() do
+    Conversation
+    |> where([c], is_nil(c.archived_at))
+    |> where(status: "closed")
+    |> where([c], c.updated_at < from_now(14, "day"))
```

Comment:
should this be `from_now(-14, "day")`? (i've never used `from_now` before 😅 )

Reply:
You are right. I replaced it with the `ago(14, "day")` query 

There are some really helpful query helpers in the [Ecto.Query.API](https://hexdocs.pm/ecto/Ecto.Query.API.html) Module :)